File: AsliMotor/Scripts/app/shared/RoleName.js
define(['jquery', 'underscore', 'backbone', 'namespace'], function ($, _, Backbone, ns) {
    ns.define('am');

    am.RoleName = {
        ADMINISTRATOR: "Administrator",
        OWNER: "Owner",
        ADMINPURCHASE: "AdminPurchase",
        ADMINSALES: "AdminSales",
        CASHIER: "Cashier",
        OWNER : "Owner",
        OWNER_ADMINSALES: "Owner, AdminSales",
        OWNER_ADMINSALES_CASHIER: "Owner, AdminSales, Cashier",
        OWNER_ADMINPURCHASE: "Owner, AdminPurchase",
        OWNER_ADMINPURCHASE_CASHIER: "Owner, AdminPurchase, Cashier",
        OWNER_ADMINISTRATOR: "Owner, Administrator"
    };

    return am.RoleName;
});

File: AsliMotor.Security/Models/RoleName.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace AsliMotor.Security.Models
{
    public class RoleName
    {
        public const string ADMINISTRATOR = "Administrator";
        public const string OWNER = "Owner";
        public const string ADMINPURCHASE = "AdminPurchase";
        public const string ADMINSALES = "AdminSales";
        public const string CASHIER = "Cashier";
        public const string OWNER_ADMINPURCHASE = "Owner, AdminPurchase";
        public const string OWNER_ADMINPURCHASE_CASHIER = "Owner, AdminPurchase, Cashier";
        public const string OWNER_ADMINSALES = "Owner, AdminSales";
        public const string OWNER_ADMINSALES_CASHIER = "Owner, AdminSales, Cashier";
    }
}
